graph-gateway/src/main.rs

```diff
@@ -147,20 +147,28 @@ async fn main() {
         legacy_signer,
     )));
 
-    // Periodically update the largest allocations for each indexer-deployment pair.
-    // TODO: Remove this once the network service is integrated and the largest allocation can be
-    //  accessed via the `Indexing` instance.
-    network
-        .indexings_largest_allocation()
-        .pipe_async(move |largest_allocations| async move {
-            let largest_allocations = largest_allocations
-                .iter()
-                .map(|(id, allocation)| ((id.indexer, id.deployment), *allocation))
-                .collect::<HashMap<_, _>>();
-
-            receipt_signer.update_allocations(&largest_allocations);
-        })
-        .forever();
+    {
```

Comment:
My idea is to remove this block, but I need to figure out how to get the `largest_allocation` address in the `client_query.rs` to release the legacy receipts from the pool.

```suggestion
    // Periodically update the largest allocations for each indexer-deployment pair.
    // TODO: Remove this once the network service is integrated and the largest allocation can be
    //  accessed via the `Indexing` instance.
    {
```